packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.tsx

```diff
@@ -117,8 +132,8 @@ const PatientSearchLaunch: React.FC<PatientSearchLaunchProps> = () => {
 
   return (
     <>
-      <div className={styles.patientSearchIconWrapper}>
-        {open && (
+      <div className={styles.patientSearchIconWrapper} ref={ref as any}>
```

Comment:
One last thing: can we get rid of the cast to `any`? We'd just have to specify the right type in place of `HTMLDivElement` on line 109, right?

Reply:
Thank you, @brandones, for the reviews